cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,7 +491,9 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
+        xit('should have appropriate class names on focus/blur', function (done) {
```

Comment:
We should not delete focus tests. Focus management is essential for an accessible UX, but it's also brittle and regressions are easily introduced when markup changes. FYI @cptvitamin